test-alg-gost3411-2012.c

```diff
@@ -65,23 +65,55 @@ test_gost2012_hash(const char *subject, const char *t, const char *match)
   return 0;
 }
 
+static int
+test_gost2012_hash512(const char *subject, const char *t, const char *match)
```

Comment:
Btw, we intend to never use 512-bit bash in gost-yescrypt.

Reply:
I know.  I just added those tests for completeness and coverage.